lib/simplifyExpression/distributeSearch/index.js

```diff
@@ -166,6 +171,36 @@ function distributeExponent(node) {
     ChangeTypes.DISTRIBUTE_EXPONENT, node, newNode, false);
 }
 
+/*
+  Given an expression in the form (base)^exponent, where base is a division node
+  and the exponent is any term, we want to distribute the exponent over each term.
+  Returns a nodeChanged Status object.
+
+  e.g. (x^2/y^2)^(2) -> x^4/y^4
+*/
+function distributeExponentOverDivision(node) {
+  // Base is required to be a division node
+  const base = getContent(node.args[0]);
+  if (!Node.Type.isOperator(base, '/')) {
```

Comment:
Yea, I know it's redundant because we call it before calling this function, but I sort of like having it in here too? Because sometimes we call functions without knowing if they will "work" or not, so I feel like it's good practice to check first.